package.json

```diff
@@ -78,7 +78,7 @@
         "@mparticle/event-models": "^1.1.6",
         "@rollup/plugin-json": "^4.1.0",
         "@rollup/plugin-typescript": "^8.3.3",
-        "@rollup/plugin-babel": "5.3.1",
+        "@rollup/plugin-babel": "6.0.3",
```

Comment:
Good call out.

I actually wonder if we can have our automated checks display a diff of the `dist` folder without having to actually commit the `dist/`. This might be helpful for our reviewing dependabot merges.

Reply:
I'm not opposed to you doing this manually for now and then following this up with a CI/CD sustained engineering ticket.